Automattic/jetpack
# Podcast Player: Add the episode selector to the sidebar

This builds on #18670, which adds the basic episode selection UI, along with the changes needed to the API.

Feedback on that was that the episode selector shouldn't live in the placeholder, and that the sidebar is the most logical place.

#### Changes proposed in this Pull Request:
This change puts the episode selector combobox in the sidebar with the other block settings.

There are still some things to iron out with the code, but it would be good to get some opinions on whether this still feels like the right approach.


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Use this branch, and add a podcast block to a post/page
* Play around with the settings and see how it feels!
* Check that swapping the RSS feed still works

#### Proposed changelog entry for your changes:


* Podcast Player: Allow the episode to be selected from the block sidebar.
